person.rb

```diff
@@ -0,0 +1,22 @@
+class Person
+  def initialize(name, age, _parent_permission)
```

Comment:
- [ ] Based on the requirements,   `name` and `parent_permission` should have default values of "Unknown" and `true.`. Kindly make the appropriate changes 👍 

### Requirements:
![image](https://github.com/Mar12358/school-library-people/assets/50990925/008b968c-694d-4eb8-a0a2-54c6556f6598)